zuul.d/tofu.yaml

```diff
@@ -6,7 +6,7 @@
       - ^ci/playbooks/molecule.*
       - ^ci_framework/playbooks/run_tofu.yml
     name: cifmw-molecule-tofu
-    nodeset: centos-9-crc-2-36-0-xl
+    nodeset: centos-9-crc-2-39-0-xl
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: openshift_login
```

Comment:
This thing is wrong, it's testing the openshift_login role :(

Reply:
https://github.com/openstack-k8s-operators/ci-framework/pull/2104

Reply:
I can see it failing if it hasn't been testing all this time so I created a new PR to fix this.